app/components/work_packages/progress/work_based/modal_body_component.sass

```diff
@@ -3,3 +3,7 @@
     background-color: unset !important
     border: none !important
     box-shadow: none !important
+
+  @media screen and (max-width: $breakpoint-sm)
```

Comment:
You are repeating the same code block here. Please use either a class or (even better) just use the [system_argumens](https://primer.style/view-components/lookbook/pages/system_arguments) for that.